.buildkite/pipelines/periodic.bwc.template.yml

```diff
@@ -6,5 +6,13 @@
           image: family/elasticsearch-ubuntu-2004
           machineType: n1-standard-32
           buildDirectory: /dev/shm/bk
+          preemptible: true
         env:
-          BWC_VERSION: $BWC_VERSION
\ No newline at end of file
+          BWC_VERSION: $BWC_VERSION
+        retry:
+          automatic:
+            - exit_status: "-1"
```

Comment:
What does this exit status represent?

Reply:
The instance just disappearing, without the agent reporting that it got a signal to exit early. It can happen for reasons other than spot termination, but I'm not sure I've ever seen it for our jobs (such as extreme resource contention to the point that the buildkite agent can't report back to the buildkite API)